languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml

```diff
@@ -3739,6 +3739,21 @@ USA
             <example correction="danificada|avariada">A minha UPS nova vinha <marker>estragada</marker>.</example>
         </rule>
 
+
+        <rule id='QUASE_QUE_PRONOME_QUASE_PRONOME' name="'quase que + pronome' → 'quase pronome'" type="style" default="temp_off">
```

Comment:
Can we shorten the rule name a little and leave it as `QUASE_QUE_PRONOME` or just `QUASE_QUE`? It's not an absolute must, but we've now got some rule writing guidelines for reference, you can check them here if it interests you 😸: https://dev.languagetool.org (scroll to the bottom of the page for PT guidelines)